core/runtime.rs

```diff
@@ -310,6 +310,10 @@ impl JsRuntime {
     state.global_context.clone().unwrap()
   }
 
+  fn v8_isolate(&mut self) -> &mut v8::OwnedIsolate {
+    self.v8_isolate.as_mut().unwrap()
+  }
```

Comment:
Don't we implement DerefMut ?

Reply:
We do, I find passing `self.v8_isolate()` more obvious than `&mut **self`